### DISCUSSION TITLE
Error: This Repository is currently Offline

### DISCUSSION THREAD

**Discussion by hassan-arif** [community/community#33318] [Repositories]
Hi everyone,

I was creating this repository ("Rat-in-a-Maz") on 15 September at around 11:30 am (UTC/GMT+5).
But as soon as I clicked "Create Repository", an error message popped saying "this repository couldn't be created".
Later on, I discovered that the repository was created. It was set public so maybe you can check it too in my profile. 

The issue is, I **can't access or delete** this repository. Every time I access its home page, there's a static message "This Repository is Currently Offline". Even the "Delete Repository" button from the settings page is gone. Also, I have not even connected any remote repository by now.

I hoped for it to resolve itself within a day or two. Unfortunately, the problem is still there and I can't do nothing about it. 
Please Help.

**Comment by radar-blip** [comment]
I looked at it. It says 'This repository is empty'.
There's nothing there.

![maz](https://user-images.githubusercontent.com/39196767/190843047-36f98e3b-76d2-4353-acd3-715b5a69dd36.png)

**Reply by hassan-arif** [reply]
From my perspective, all I see is the following message in this repository:
![1](https://user-images.githubusercontent.com/102779695/190853253-330540eb-d3e3-424b-9d18-b4ec008fbedd.JPG)
This only appears in this repository, others appear fine.

Also, the option to delete the repository is also unavailable.
![2](https://user-images.githubusercontent.com/102779695/190853256-c4f7afdb-fff1-414b-9cdd-50b135efc1d6.JPG)
Even if I try to modify some settings then they are always unsuccessful.

Here I've even tried to remote link this repository and upload files from bash but it responds with "No repository found"
![3](https://user-images.githubusercontent.com/102779695/190853257-3afd463f-ac81-49ed-b60f-95cb26d91558.JPG)
![4](https://user-images.githubusercontent.com/102779695/190853259-36e18a4d-0d69-4923-819d-76cb87587d92.JPG)

**Reply by airtower-luna** [reply]
I see the same thing as @linuxlawson, so maybe it's a regional datacenter that's broken or something? Anyway, in that case you should do as the error message says and [contact GitHub support](https://support.github.com/request), us other users can't help there. I hope it gets fixed soon!
